src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -97,7 +99,7 @@ public ServerCubeCache(ICubicWorldServer worldServer) {
 		this.columnGenerator = new ColumnGenerator(worldServer);
 		this.loadedColumns = Maps.newHashMap();
 		this.cubesToUnload = new ArrayDeque<>();
-		this.forceAdded = new HashMap<>();
+		this.forceAdded = new HashMap<>();		
```

Comment:
Could you avoid these kind of whitespace changes? Expecially since I have IntelliJ IDEA formatting setting file in the repository (I know your IDE probably doesn't show trailing wwhitespace)